source/traction_boundary_conditions/initial_lithostatic_pressure.cc

```diff
@@ -0,0 +1,441 @@
+/*
+  Copyright (C) 2011 - 2016 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/traction_boundary_conditions/initial_lithostatic_pressure.h>
+#include <aspect/global.h>
+#include <aspect/utilities.h>
+#include <deal.II/base/std_cxx11/array.h>
+
+#include <aspect/geometry_model/sphere.h>
+#include <aspect/geometry_model/spherical_shell.h>
+#include <aspect/geometry_model/chunk.h>
+//#include "layered_chunk.h"
```

Comment:
I guess we have to wait until the `layered_chunk` is merged before this one can be finished...